static/js/schema-impl/capability/thing.js

```diff
@@ -301,9 +301,9 @@ class Thing {
           const action = description.actions[name];
 
           let href;
-          for (const link of description.actions[name].links) {
-            if (link.rel === 'action') {
-              href = link.href;
+          for (const form of description.actions[name].forms) {
+            if (!form.op || form.op === Constants.WoTOperation.INVOKE_ACTION) {
+              href = form.href;
```

Comment:
In contrast to the above I think this is probably safe since it should pick the form generated by the gateway which is the last in the array and contains no op member.